---
title: "RNAfusion"
format: html
toc: true
number-sections: true
---

# Introduzione

**RNAfusion** Ã¨ una [pipeline nf-core](https://nf-co.re/) per l'analisi di fusioni di RNA-seq.  
Fornisce un workflow standardizzato, riproducibile e facilmente estendibile, costruito con [Nextflow](https://www.nextflow.io/) e containerizzato tramite Docker/Singularity.

---

# Caratteristiche principali

- Multipiattaforma e portabile  
- Gestione container con Docker/Singularity  
- Integrazione con ambienti HPC e cloud  
- Report interattivi in HTML  
- Test e validazione automatizzati con `nf-core lint`

---

# Installazione

<h1>
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="docs/images/nf-core-rnafusion_logo_dark.png">
    <img alt="nf-core/rnafusion" src="docs/images/nf-core-rnafusion_logo_light.png">
  </picture>
</h1>
Per installare ed eseguire la pipeline:

```bash
nextflow run nf-core/rnafusion -profile test,docker



## Citations

If you use nf-core/rnafusion for your analysis, please cite it using the following doi: [10.5281/zenodo.3946477](https://doi.org/10.5281/zenodo.3946477)

An extensive list of references for the tools used by the pipeline can be found in the [`CITATIONS.md`](CITATIONS.md) file.

You can cite the `nf-core` publication as follows:

> **The nf-core framework for community-curated bioinformatics pipelines.**
>
> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen.
>
> _Nat Biotechnol._ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).



## KALLISTO fusions
kallisto can now find reads which span potential fusion breakpoints. The quant mode adds a --fusion flag which identifies read pairs involved in fusions and writes output to fusion.txt, this file is then processed by pizzly for downstream analysis.